.github/workflows/docker_ci.yml

```diff
@@ -44,16 +44,36 @@ jobs:
           env
           echo "C Compiler:"
           gcc --version
-
       - name: Add dependencies
-        run: |
-          apk add libc6-compat
-
+        run: apk add libc6-compat
       - name: Build V
         run: CC=gcc make
-
       - name: All code is formatted
         run: VJOBS=1 ./v test-cleancode
-
       - name: Run only essential tests
         run: VTEST_JUST_ESSENTIAL=1 ./v test-self
+
+  ubuntu-musl:
```

Comment:
```suggestion
  ubuntu-docker-musl:
```

Reply:
The longer job name makes it easier to search and identify.
The normal runner uses `ubuntu` too, and that environment is different than the the one from `thevlang/vlang:ubuntu-build` .